### ISSUE TITLE
Accessibility: Focus indicator missing on some buttons and form controls

### ISSUE THREAD

**Issue by kemiga** [sky-uk/toolkit-ui#145]
## Expected behaviour
All interactive elements should receive a visible focus to enable keyboard users to determine their location on a page.

## Current behaviour
Currently, some buttons, the “Select” checkboxes the “Radio” buttons, and the radio buttons in the "Form" do not have a focus indicator.

## Suggestion
A highly visible focus indicator, e.g. a border, should be provided for all interactive elements.

**Comment by joebell93** [comment]
@kemiga this is currently in progress and will be in the next release!

Also the latest checkbox/radio buttons do have focus indicators, make sure you're on the latest version.
